src/components/ThymeTable/Entry.js

```diff
@@ -56,7 +61,10 @@ type EntryStateType = {
 class Entry extends Component<EntryType, EntryStateType> {
   constructor(props: EntryType) {
     super(props);
-
+    const { settings } = this.props;
+    const { rounding, roundingDown } = settings;
+    this.rounding = rounding;
+    this.roundingDown = roundingDown;
```

Comment:
I think it's better to consult `this.props.settings.rounding` directly when needed instead of binding the value to a property on the component. This prevents polluting the component with values it already has.